browser/brave_content_browser_client.cc

```diff
@@ -129,11 +130,13 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #if BUILDFLAG(BRAVE_WALLET_ENABLED)
 #include "brave/browser/brave_wallet/brave_wallet_context_utils.h"
 #include "brave/browser/brave_wallet/brave_wallet_service_factory.h"
+#include "brave/browser/ui/webui/brave_wallet/wallet_panel_ui.h"
```

Comment:
these should also be in `!defined(OS_ANDROID)` to match gn and usage below

Reply:
Fixed in 69f8b573f8508373220d3f1f4ae9578ac065d6c9